docker/base_cuda_10.dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM stitchem/stitchem-base:latest
+
+ENV CUDA=10.1.105-1
+ENV CUDA_APT=10-1
+ENV CUDA_SHORT=10.1
+ENV CUDA_INSTALLER=cuda-repo-ubuntu1804_${CUDA}_amd64.deb
+
+RUN wget http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/${CUDA_INSTALLER}
```

Comment:
Are the RUN lines identical for all CUDA versions?

If so, couldn't it be something like 
```
FROM stitchem/stitchem-base:latest

ENV CUDA=10.1.105-1
ENV ...

FROM cuda-install
```

Reply:
I tried to factorize, until I realized I needed gcc-5 to compile with CUDA 8, so I figured we could leave it like that